docs/cloud/data-ingestion.md

```diff
@@ -21,3 +21,83 @@ If your Graphite stack does not currently contain any relay, then you can simply
 
 When creating a Hosted Metrics Graphite instance, we provide a carbon-relay-ng config file that you can plug in and be ready to use out of the box.
 We also have Grafana Labs engineers ready to advise further on set up, if needed.
+
+## Using carbon-relay-ng as a replacement for carbon-relay or carbon-cache
+
+The most simple way to send your carbon traffic to GrafanaCloud is to use carbon-relay-ng as a replacement for your current carbon-relay or carbon-cache. Carbon-relay-ng has a carbon input which supports the plain text and the pickle protocols, just like carbon-relay and carbon-cache. 
```

Comment:
at this time, crng's CH implementation behaves differently from the python code.
see https://github.com/grafana/carbon-relay-ng/pull/336
so if people were to swap in, it would shuffle data around.

Reply:
Good point, I'll add a note for that